release-please-config.json

```diff
@@ -0,0 +1,84 @@
+{
+  "bootstrap-sha": "1cef37c5d8c7d97ee52a8e6b71f12150063b8503",
+  "separate-pull-requests": true,
+  "release-type": "python",
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "include-component-in-tag": true,
+  "tag-separator": "/",
+  "packages": {
+    "providers/openfeature-provider-flagd": {
+      "package-name": "providers/flagd",
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "versioning": "default",
+      "extra-files": [
+        "README.md"
+      ]
+    },
+    "hooks/openfeature-hooks-opentelemetry": {
+      "package-name": "hooks/opentelemetry",
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "versioning": "default",
+      "extra-files": [
+        "README.md"
+      ]
+    }
+  },
+  "changelog-sections": [
+    {
+      "type": "fix",
+      "section": "🐛 Bug Fixes"
+    },
+    {
+      "type": "feat",
+      "section": "✨ New Features"
+    },
+    {
+      "type": "chore",
+      "section": "🧹 Chore"
+    },
+    {
+      "type": "docs",
+      "section": "📚 Documentation"
+    },
+    {
+      "type": "perf",
+      "section": "🚀 Performance"
+    },
+    {
+      "type": "build",
+      "hidden": true,
+      "section": "🛠️ Build"
+    },
+    {
+      "type": "deps",
+      "section": "📦 Dependencies"
+    },
+    {
+      "type": "ci",
+      "hidden": true,
+      "section": "🚦 CI"
+    },
+    {
+      "type": "refactor",
+      "section": "🔄 Refactoring"
+    },
+    {
+      "type": "revert",
+      "section": "🔙 Reverts"
+    },
+    {
+      "type": "style",
+      "hidden": true,
+      "section": "🎨 Styling"
+    },
+    {
+      "type": "test",
+      "hidden": true,
+      "section": "🧪 Tests"
+    }
+  ],
```

Comment:
TODO: we should add these sections to the python-sdk repo config, to standarize both changelogs.

Reply:
I'd say yes, it makes sense. They are also used in the go, js, and java sdks